packages/perseus/src/widgets/radio/choice.jsx

```diff
@@ -272,6 +272,8 @@ class Choice extends React.Component<$FlowFixMe, State> {
                             this.props.pos,
                         )}`}
                         aria-selected={this.props.checked}
+                        aria-checked={this.props.checked}
+                        role={"checkbox"}
```

Comment:
**question:** I'm surprised this didn't have an effect on the tests.  Is it possible that those tests are still passing when checking for the `button` role due to the second `<Clickable>` below that's wrapped in the popover?